short-response.md

```diff
@@ -93,7 +119,7 @@
   shoutOut();
   console.log(`The best designer in the room is ${theCreator}`.);
   ```
-
+  - This logs `The best designer in the room is Peter` because `theCreator` was properly reassiged by the invokation of the `shoutOut` function that could access it because scope allows children scope to access parent ones.
```

Comment:
# Question 10 Score:
## Technical
**3**: Solid

## Writing
**2**: A couple of typos. But otherwise very clear and concise